docs/src/main/paradox/getting-started/running-cluster.md

```diff
@@ -0,0 +1,95 @@
+# Running the Projection in Akka Cluster
+
+Running the Projection with [Akka Cluster](https://doc.akka.io/docs/akka/current/typed/cluster.html) allows us to add two important aspects to our system: availability and scalability.
+A Projection running as a single Actor creates a single point of failure (availability), when the app shuts down for any reason, the projection is no longer running until it's started again.
+A Projection running as a single Actor creates a processing bottleneck (scalability), all messages from the @apidoc[SourceProvider] are processed by a single Actor on a single machine.
+By using a [Sharded Daemon Process](https://doc.akka.io/docs/akka/current/typed/cluster-sharded-daemon-process.html#sharded-daemon-process) with Akka Cluster and [Akka Cluster Sharding](https://doc.akka.io/docs/akka/current/typed/cluster-sharding.html) we can scale up the Projection and make it more available by running as many instances of the same Projection as we have cluster members.
```

Comment:
Maybe change "as many instances" to "at least as many instances". Since the number of instances is a fixed number that is not easy to change the recommendation should be to use more instances so there is room for scaling out in the future.

Reply:
:+1: